tests/test_meta.py

```diff
@@ -0,0 +1,38 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+import os
+import pkgutil
+import subprocess
+import sys
+import typing
+
+import cryptography
+
+
+def find_all_modules() -> typing.List[str]:
+    return sorted(
+        mod
+        for _, mod, _ in pkgutil.walk_packages(
+            cryptography.__path__,  # type: ignore[attr-defined]
+            prefix=cryptography.__name__ + ".",
+        )
+    )
+
+
+def test_no_circular_imports(subtests):
+    env = os.environ.copy()
+    env["PYTHONPATH"] = os.pathsep.join(sys.path)
+
+    # When using pytest-cov it attempts to instrument subprocesses. This
+    # causes the memleak tests to raise exceptions.
+    # we don't need coverage so we remove the env vars.
+    env.pop("COV_CORE_CONFIG", None)
+    env.pop("COV_CORE_DATAFILE", None)
+    env.pop("COV_CORE_SOURCE", None)
```

Comment:
Dunno if it's preferable but the same can be achieved with the built-in `monkeypatch` fixture: https://docs.pytest.org/en/latest/monkeypatch.html.

Reply:
We don't want to mutate the parent process's env, we only want to mess with the child processes.